### PR TITLE
add verify-connector-node-metadata namespace

### PR DESCRIPTION
**Pull request by philandstuff**
as requested by @Wynndow

**!!! Merge alphagov/verify-metadata#334 first !!!**

### CODE REVIEW

**Comment by Krenair** [comment]
There is no ci/verify path on verify-metadata.git master branch.

**Comment by philandstuff** [comment]
Yes, @Wynndow said IRL yesterday that he plans to do that next

**Comment by Krenair** [comment]
👍 lets do that first?

**Comment by Wynndow** [comment]
This is the PR to add that path, will let you know when it's merged: https://github.com/alphagov/verify-metadata/pull/334

File: values.yaml [hunk 1]
```diff
@@ -23,6 +23,13 @@ httpsEgressSafelist:
 namespaces:
 - name: verify-main
 
+- name: verify-connector-node-metadata
```
Comment (CharlesIC) [inline values.yaml]:
```suggestion
- name: connector-node-metadata
```
This is the _verify_ cluster so we can call the new namespace just `connector-node-metadata` - just like we have `proxy-node-build`, etc

File: values.yaml [hunk 2]
```diff
@@ -23,6 +23,13 @@ httpsEgressSafelist:
...
+  owner: alphagov
+  repository: verify-metadata
+  branch: master
+  path: ci/verify
+  requiredApprovalCount: 2
```
Comment (CharlesIC) [inline values.yaml]: `requiredApprovalCount: 2`
@philandstuff Can we remove this for now? A lot of Verify devs push to this repo on a weekly basis and only a handful have been added to the [GSP trusted developers repo](https://github.com/alphagov/gds-trusted-developers/tree/master/users)

File: values.yaml (hunk 1 above)
Comment (CharlesIC) [reply values.yaml]: This is OK, ignore this comment

**Pull request merged**